//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// aceg_T_A.h
//
// Code generation for function 'aceg_T_A'
//

#ifndef ACEG_T_A_H
#define ACEG_T_A_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void aceg_T_A(double V_init, double A_init, double A_wayp, double V_max,
              double J_max, double J_min, double T, double t[7]);

#endif
// End of code generation (aceg_T_A.h)
